### ISSUE TITLE
remember new transaction type

### ISSUE THREAD

**Issue by sedrubal** [mtotschnig/MyExpenses#992]
Most of the time, we're using split transactions. Therefore, it would help a lot of the last transaction type (normal transaction, transfer, split transaction) was be saved for the next time and the FAB button / the big ➕ on the transaction list automatically choose the same type of transaction again. Currently, we always have to tap on  the ➕ button and switch to "new split".

**Comment by mtotschnig** [comment]
There is an alternative way of creating a new split, that is a bit speedier than the method you describe: Long press on the FAB for opening a submenu, which has "New split".
This still is an interesting idea.

**Comment by sedrubal** [comment]
Yes, I also found this feature of long pressing on the FAB. But it would still speed up the process of entering transactions if it was just one tab and it makes things easier for us because you don't forget to switch every time if you forgot to choose new split in the submenu ;)
